File: domain/src/main/java/no/nav/melosys/domain/brev/storbritannia/Utsendelse.java
package no.nav.melosys.domain.brev.storbritannia;

import java.time.LocalDate;
import java.util.List;

import no.nav.melosys.domain.kodeverk.lovvalgsbestemmelser.Lovvalgbestemmelser_trygdeavtale_uk;

public record Utsendelse(
    Lovvalgbestemmelser_trygdeavtale_uk artikkel,
    List<String> oppholdsadresseUK,
    LocalDate startdato,
    LocalDate sluttdato
) {
}

File: domain/src/main/java/no/nav/melosys/domain/brev/trygdeavtale/Utsendelse.kt
package no.nav.melosys.domain.brev.trygdeavtale

import no.nav.melosys.domain.kodeverk.LovvalgBestemmelse
import java.time.LocalDate

@JvmRecord
data class Utsendelse(
    val artikkel: LovvalgBestemmelse?,
    val oppholdsadresse: MutableList<String?>?,
    val startdato: LocalDate?,
    val sluttdato: LocalDate?
)
